frontend/src/components/layout/Breadcrumb.tsx

```diff
@@ -0,0 +1,60 @@
+import { useRouterState, Link } from '@tanstack/react-router';
+import { ChevronRight } from 'lucide-react';
+
+function Breadcrumb() {
+  const pathNameMap: { [key: string]: string } = {
+    '/': 'Dashboard',
+    '/sensors': 'Sensoren',
+    '/settings': 'Einstellungen',
+    '/team': 'Mitarbeitende',
+    '/treecluster': 'Baumgruppen',
+    '/vehicles': 'Fahrzeuge',
+    '/waypoints/': 'Einsatzpläne',
+    '/waypoints/new': 'Neuer Einsatzplan',
+    '/map/': 'Kataster',
+  };
+
+  const matches = useRouterState({ select: (s) => s.matches });
+  const breadcrumbs = matches
+    .filter(({ pathname }) => pathname !== '/')
+    .map(({ pathname }) => {
+      const title = pathNameMap[pathname] || 'Kein Titel vorhanden';
+      return {
+        title,
+        path: pathname,
+      };
+    });
```

Comment:
Maybe we should wrap this in a `useMemo` so it doesn't have to be recalculated on every rerender. It's only dependent on `matches`.

```suggestion
  const breadcrumbs = useMemo(() => matches
    .filter(({ pathname }) => pathname !== '/')
    .map(({ pathname }) => {
      const title = pathNameMap[pathname] || 'Kein Titel vorhanden';
      return {
        title,
        path: pathname,
      };
    }), [matches]);
```